metricflow/test/fixtures/semantic_manifest_yamls/simple_manifest/saved_queries.yaml

```diff
@@ -0,0 +1,12 @@
+---
+saved_query:
+  name: p0_booking
+  description: Booking-related metrics that are of the highest priority.
+  metrics:
+    - bookings
+    - instant_bookings
+  group_bys:
+    - TimeDimension('metric_time', 'day')
+    - Dimension('listing__capacity_latest')
+  where:
```

Comment:
@plypaul I think this should be `filter` to keep it consistent with metrics.

Reply:
@Jstein77 we decided that `where` was what we'd use for saved queries because they're queries, and `where` matches the query interfaces for the CLI, JDBC and GraphQL.

`filter` is what we use for metrics and other non-query objects in the spec. 

I'd prefer to pick one and use it everywhere, but alas that ship has sailed. Or we can go with `filter` here and commit to changing the other query interfaces.

One argument in favor of `where` - we may need to introduce a post-aggregation filter (a `having` input) for query time specification and that can be a natural distinction for people accustomed to SQL. That said, I don't like calling anything `having` because it's a goofy name for a filter expression.